### PR TITLE
A big overhaul of some stuff

### PR DESCRIPTION
**Pull request by OfficialTCGMatt**
Okay, when TCG decides to take over some, he decides it's a great idea to make things his way, so pls no get mad :scream:
Changed a lot of stuff :P

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+};
+
+// Cool Error Handler Script
+// Provided by Yours Truly, TCG
+// Version 1.0.0.0
+// Modified 12/24/2020
+
+meowos.ErrorHandler = function(location = "SYSTEM", error, reportToDevs = false) {
+  console.log(error);
+  document.body.innerHTML =
```
Comment (itzTheMeow) [inline script.js:40]: should probably stop executing scripts, so it doesnt crash the rest of the scripts and possibly break something more :3

File: server-local.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 const app = require("./express/server");
-
-app.listen(3000, () => console.log("Local app listening on port 3000!"));
+const port = 3000; /* Be changeable? */
```
Comment (itzTheMeow) [inline server-local.js:4]: i believe netlify requires port 3000 to work with functions

**Review by itzTheMeow — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (OfficialTCGMatt) [reply script.js:40]: k

File: server-local.js (hunk 2 above)
Comment (OfficialTCGMatt) [reply server-local.js:4]: :RengeShrug:

**Review by itzTheMeow — APPROVED** [review]

**Pull request merged**